src/OrbitBase/include/OrbitBase/GetProcessIds.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_GET_PROCESS_IDS_H
```

Comment:
Just a heads-up that the styleguide want the trailing underscore in the include guards. We do it quite consistently.

Reply:
oh, right, done.